spec/support/fetchable_behavior.rb

```diff
@@ -115,4 +115,16 @@
       it { is_expected.to eq true }
     end
   end
+
+  describe '#indexed?' do
+    subject(:indexed) { record.indexed? }
+
+    context 'when the last_crawl_status = "OK"' do
```

Comment:
Minor nit: it might be nice to have a test for when `#indexed?` returns false.

Reply:
can do